src/app/containers/Rewards/saga.ts

```diff
@@ -315,10 +319,13 @@ export function* deposit(action: { type: string; payload: DepositPayload }) {
       );
     }
     yield put(RewardsActions.setIsDepositing(false));
+    toast.success("deposit successful");
+    yield put(GlobalActions.getTokenBalances());
+    yield put(RewardsActions.getRewardPoolsData(pools));
   } catch (e: any) {
     console.log(e);
     if (e.code === -32603) {
```

Comment:
lets avoid magic numbers, please consider placing this `error code` where it can easily be identify and replaced. e.g
```suggestion
    if (e.code ===  Errors.withdraw) {
```